daml-foundations/daml-ghc/package-database/BUILD.bazel

```diff
@@ -123,9 +124,25 @@ daml_package_db(
   visibility = ["//visibility:public"]
 )
 
+daml_package_db(
+  name = "package_db_for_damlc",
+  pkgs =  [":daml-prim-1.0"
```

Comment:
Could we have a list of DAML-LF versions somewhere in this file and generate this of packages list from the list of versions? We could also reuse the list of versions a bit further below.